src/cmd/linuxkit/registry/manifest.go

```diff
@@ -0,0 +1,114 @@
+package registry
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"strings"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"github.com/estesp/manifest-tool/pkg/registry"
+	"github.com/estesp/manifest-tool/pkg/types"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	notaryServer                     = "https://notary.docker.io"
+	notaryDelegationPassphraseEnvVar = "NOTARY_DELEGATION_PASSPHRASE"
+	notaryAuthEnvVar                 = "NOTARY_AUTH"
+	dctEnvVar                        = "DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE"
+)
+
+var platforms = []string{
+	"linux/amd64", "linux/arm64", "linux/s390x",
```

Comment:
I don't like hard coding this, don't have to change now but its a bit of a design smell.

Reply:
I don't either, but it was there before. If we can come up with a good way, we can do a dedicated PR for it.